QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
+QUEST_20150714_002179	$Klaipeda Market
+QUEST_20150714_002180	$Give me the effigy. I shall calm the spirits.
+QUEST_20150714_002181	$The monsters will feel threatened by the energies that spurt out from the crystals containing my authority and begin their attack.
+QUEST_20150714_002182	$Please protect the crystals so that they may produce my authority and spread it to Sirdgelos Forest.
+QUEST_20150714_002183	$You prepared this, brother? You're truly a great man.
+QUEST_20150714_002184	$I can sense the wrath of the spirits.
+QUEST_20150714_002185	$Incense Burner
+QUEST_20150714_002186	$The scent of the Incense Burner is spreading.
+QUEST_20150714_002187	$I can feel the spirits are complaining.
+QUEST_20150714_002188	$I can feel the pain of the spirits.
+QUEST_20150714_002189	$I can feel the regrets of the spirits.
+QUEST_20150714_002190	$The seal is not yet ready? {nl}It pains me so that you are the only one I can trust.
+QUEST_20150714_002191	$Can you solve the complaints of the villagers' spirits in Seven Valley?
+QUEST_20150714_002192	$The Basics of Ignition
+QUEST_20150714_002193	$Ignition differs by elements, but with the blessing from the Goddess, there may be cases where we can ignore that. {nl}To do so, we should try hard to achieve the scholarly knowledge and the faithfulness to the Goddesses.
+QUEST_20150714_002194	$1. Dried environment{nl}2. Something that can be ridden on. {nl}3. Enough mana. {nl}4. The blessing of the Goddesses.
+QUEST_20150714_002195	$Flurry… turned into a handful of ashes by illuminating a great light. {nl}Did that demon do something? {nl}I had no idea, even as a probationary wizard.
+QUEST_20150714_002196	$[Caution]{nl}- It is the last stage of the experiment. Be cautious when handling it -
+QUEST_20150714_002197	$1. Do not overreact Schilt Essences into a sample of Petrification Detector. {nl}2. Do not keep the Schilt Amplifier activated for a long time. {nl}3. Even if you become immune to petrification using the Schilt Amplifier, it will only be temporary so do not stay in the lab room for too long.
+QUEST_20150714_002198	$[Usage Limit]{nl}- Due to a recent incident, we sealed the isolated lab room. {nl}Only people permitted by the chief can enter the room via the portal.
+QUEST_20150714_002199	$[Method of Usage]{nl}1. Use the Schilt Essences on the portal generator. {nl}2. One person can enter after a certain number of Essences are used. {nl}3. When more than one person enters simultaneously, please notify and ask the chief.
+QUEST_20150714_002200	$Petrification in Progress
+QUEST_20150714_002201	$My body is becoming stiff.
+QUEST_20150714_002202	$We can move faster.
+QUEST_20150714_002203	$We tried our best to impede them until your arrival. {nl}But as you can see, we're at our limit.
+QUEST_20150714_002204	$I've received your letter. {nl}Now is the chance the Goddesses gave to you since Legwyn is dead. {nl}It's great to hear that the lads of Legwyn family trusts you.
```

Comment:
"It's great to hear that the lads of Legwyn family **trust** you."
or
"It's great to hear that the **lad** of Legwyn family trusts you."